### PR TITLE
Doc update for Orin Nano BOM

### PR DESCRIPTION
**Pull request by tokk-nv**
Document updated.
Staged: https://tokk-nv.github.io/jetbot/master/bill_of_materials_orin.html

### CODE REVIEW

File: docs/bill_of_materials_orin.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Bill of Materials - Jetson Orin Nano version
+
+This page lists all the parts you need to build the open-source JetBot with the new [**Jetson Orin Nano 8GB Developer Kit**](https://developer.nvidia.com/embedded/learn/get-started-jetson-orin-nano-devkit), along with purchasing links from popular vendors.
```
Comment (jaybdub) [inline docs/bill_of_materials_orin.md:3]: Maybe add a note here about when to use Jetson Orin Nano, and about Jetson Nano EOL

File: docs/bill_of_materials_orin.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+### Common parts
+
+You need these components for each JetBot.
+
+|  **Part** | **Quantity** | **Cost** | **URL** | **Notes** |
+| --- | --: | --: | --- | --- |
+| Jetson Orin Nano 8GB Developer Kit | 1 |  $499.00 | [NVIDIA](https://store.nvidia.com/jetson/store/)  |  |
+| Micro SD card | 1 | $14.95 | [Amazon](https://a.co/d/0aOqPQh) | 128GB |
+| Battery | 1 | $12.99 | [Amazon](https://a.co/d/5DvsXGu) | PD 20W output, 10,000mAh |
+| USB-C PD cable | 1 | $10.99 | [Amazon](https://a.co/d/ihjA431) | USB-C to DC 5.5mm x 2.5mm, right angle |
```
Comment (jaybdub) [inline docs/bill_of_materials_orin.md:16]: Not an issue, but do they have a smaller version of this cable?

**Review by jaybdub — CHANGES_REQUESTED** [review]
Looks good to me, just pending testing / a couple comments.

File: docs/bill_of_materials_orin.md (hunk 1 above)
Comment (tokk-nv) [reply docs/bill_of_materials_orin.md:3]: Good idea, will do.

File: docs/bill_of_materials_orin.md (hunk 2 above)
Comment (tokk-nv) [reply docs/bill_of_materials_orin.md:16]: I have searched around and evaluated alternatives (such as shown in the photo below), but did not have a good minimum version of the cable.
I will test to see if I can cut and solder them internal cables to effectively shorten the cable, and if that is the case I will document for those people prefer the nice tight cable management.

IMG_5918

Comment (jaybdub) [reply docs/bill_of_materials_orin.md:16]: I think it's alright, was just wondering.  Soldering the internal cables is probably too much to ask developers.

**Review by jaybdub — APPROVED** [review]
LGTM

**Pull request merged**